app/jobs/regular/discourse_reactions/post_action_synchronizer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
```

Comment:
hmmm how do we kick off the "first" sync? Do we just create a scheduled job that checks if we are behind on a sync?